docs/compounds/result-card.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+</head>
+
+<body>
+  <div class="container">
+    <h1>Result Card</h1>
+    <div class="row">
+      <div class="review-item col-md-4 d-inline-block align-top">
+        <figure class="w-100 m-0">
+          <picture class="d-block position-relative">
```

Comment:
once you moved this up please also check the behaviour by removing the classes. I don't think we need of them